learn/core_concepts/indexes.md

```diff
@@ -73,7 +73,7 @@ Since synonyms are linked to a given index, they won't apply to any other index
 
 ## Stop words
 
-Sometimes you may want to ignore certain words in documents and search queries. To do so, **a set of stop words can be defined for an index**. Unless you actually need them, some words neither add semantic value nor context. Besides, they are often too frequent (i.e., `the` or `of` in English).
+Sometimes you may want to ignore certain words in documents and search queries. To do so, **a set of stop words can be defined for an index**. Unless you actually need them, some words neither add semantic value nor context. Besides, they are often too frequent (i.e., `the`, `or`, `of` in English).
```

Comment:
Although I agree `or` is a good stop word in English, this one actually means that either one of the listed words (i.e. `the` and `or`) are words users should consider adding to the stopword list.